### PR TITLE
WM4-Nagehan-Uyumaz-Full-Stack-Project-Assessment

### PR DESCRIPTION
**Pull request by Nagehan1**

### CODE REVIEW

**Review by selinfildis — COMMENTED** [review]
Heya 👋 

Few general comments: 
- Could you use a formatter to format the JSX components. The indentation etc seems a bit off. have you used prettier to format it?
- the `companents` directory seems to be named wrong. should be `components` :) 
could use: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker to ensure you don't have typos :) 
- Could you include a view of the UI for level 100 to the PR please :)

File: client/src/companents/Rating.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+
+function Rating(props) {
+const[rating,setRating]=useState(props.id.rating)
+console.log(rating)
+const handleIncrease=()=>{
+    setRating(rating+1);
+    
+}
+const handleDicrease=()=>{
```
[the remark this answers is not in the document]
Comment (Nagehan1) [reply client/src/companents/Rating.jsx]: > thank you

File: client/src/companents/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react'
+
+import Videocart from './Videocart'
+import dataVideos from "/Users/nagehan/Documents/Full-Stack-Project-Assessment/client/src/exampleresponse.json"
```
[the remark this answers is not in the document]
Comment (Nagehan1) [reply client/src/companents/Main.jsx]: In this step we were asked to use static data.

File: client/src/App.js [hunk 3]
```diff
@@ -1,19 +1,11 @@
 import React from "react";
 import "./App.css";
-import Video from "./Video";
-import dataVideos from "./exampleresponse.json";
+import Main from "./companents/Main";
```
[the remark this answers is not in the document]
Comment (Nagehan1) [reply client/src/App.js:3]: I will re-think about it.Thank you:)

[the remark this answers is not in the document]
Comment (Nagehan1) [reply client/src/App.js:3]: Thank you for review Selin .It was very helpful.

**Pull request closed without merging**